enclave-manager/web/src/emui/enclaves/enclave/service/logs/ServiceLogs.tsx

```diff
@@ -45,7 +45,8 @@ export const ServiceLogs = ({ enclave, service }: ServiceLogsProps) => {
         if (cancelled) {
           return;
         }
-        setLogLines((logLines) => [...logLines, { message: `Error: ${stringifyError(error)}`, status: "error" }]);
+        // TODO: We need a way to show this error
+        alert (`Error: ${stringifyError(error)}`)
```

Comment:
Easiest option that's nicer than `alert` would be to use a `toast`. Slightly nicer would be to store the message in state called `error` and use a `KurtosisAlert` component under/above the log viewer to display the error.

Reply:
Agreed and I thought about the toast but we don't actually want a toast. We need to auto-reconnect and not bother the user. As mentioned in the PR text we need to do more work here. It's just temporary to surface the error.